Bulldozer.CSV/Maps/Person.cs

```diff
@@ -0,0 +1,911 @@
+
+                    case Rock.Model.Gender.Unknown:
+                        newPerson.Gender = Rock.Model.Gender.Unknown.ConvertToInt();
+                        break;
+                }
+
+                if ( !string.IsNullOrEmpty( personCsv.MaritalStatus ) )
+                {
+                    newPerson.MaritalStatusValueId = MaritalStatusDVDict[personCsv.MaritalStatus]?.Id;
+                }
+
+                // do a case-insensitive lookup GradeOffset from either the Description ("Kindergarten", "1st Grade", etc) or Abbreviation ("K", "1st", etc)
+                int? gradeOffset = null;
+                if ( !string.IsNullOrWhiteSpace( newPerson.Grade ) )
+                {
+                    gradeOffset = gradeOffsetLookupFromDescription.GetValueOrNull( newPerson.Grade );
+                    if ( gradeOffset == null )
+                    {
+                        gradeOffset = gradeOffsetLookupFromAbbreviation.GetValueOrNull( newPerson.Grade );
+                    }
+                }
+
+                if ( gradeOffset.HasValue )
+                {
+                    newPerson.GraduationYear = Person.GraduationYearFromGradeOffset( gradeOffset );
+                }
+
+                switch ( personCsv.EmailPreference )
+                {
+                    case EmailPreference.EmailAllowed:
+                        newPerson.EmailPreference = EmailPreference.EmailAllowed.ConvertToInt();
+                        break;
+
+                    case EmailPreference.DoNotEmail:
+                        newPerson.EmailPreference = EmailPreference.DoNotEmail.ConvertToInt();
+                        break;
+
+                    case EmailPreference.NoMassEmails:
+                        newPerson.EmailPreference = EmailPreference.NoMassEmails.ConvertToInt();
+                        break;
+                }
+
+                personImportList.Add( newPerson );
+            }
+
+            var importDateTime = RockDateTime.Now;
+
+
+            int familyChildRoleId = familyRolesLookup[Rock.SystemGuid.GroupRole.GROUPROLE_FAMILY_MEMBER_CHILD.AsGuid()].Id;
+            var personAliasesToInsert = new List<PersonAlias>();
+
+            foreach ( var personImport in personImportList )
+            {
+                Person person = null;
+                if ( personLookup.ContainsKey( personImport.PersonForeignKey ) )
+                {
+                    person = personLookup[personImport.PersonForeignKey];
+                }
+                else if ( personImport.PreviousPersonIds.Count > 0 )
+                {
+                    var previousForeignIds = personImport.PreviousPersonIds.Select( i => string.Format( "{0}^{1}", ImportInstanceFKPrefix, i ) );
+                    person = personLookup.FirstOrDefault( d => previousForeignIds.Any( i => i == d.Value.ForeignKey ) ).Value;
+                    if ( person != null )
+                    {
+                        personAliasesToInsert.Add( new PersonAlias
```

Comment:
Should iterate through all pereviousForeignIds and create aliases for each, using the foreignId as AliasPersonId